crates/bevy_reflect/bevy_reflect_derive/src/type_uuid.rs

```diff
@@ -15,7 +15,9 @@ pub fn type_uuid_derive(input: proc_macro::TokenStream) -> proc_macro::TokenStre
 
     // Build the trait implementation
     let name = &ast.ident;
-    let (impl_generics, type_generics, where_clause) = &ast.generics.split_for_impl();
+    let (_impl_generics, _type_generics, _where_clause) = &ast.generics.split_for_impl();
+
+    // TODO: panic if there are generics.
```

Comment:
I would appreciate help here as how to check if a generic exists and then `panic!`ing for better error messages.